packages/polaris-viz/src/components/BarChart/stories/Default.stories.tsx

```diff
@@ -8,8 +8,63 @@ import {DEFAULT_DATA, Template} from './data';
 
 export const Default: Story<BarChartProps> = Template.bind({});
 
+const data = [
+  {
+    name: 'Monday',
```

Comment:
Easier to reason about data structure where the top level `name` would be the primary dimension/group and then the `series.data` array are the bars.